src/game/Spells/SpellAuras.cpp

```diff
@@ -952,6 +952,7 @@ bool Aura::CanProcFrom(SpellEntry const* spell, uint32 EventProcEx, uint32 procE
     uint64 mask = sSpellMgr.GetSpellAffectMask(GetId(), GetEffIndex());
 
     // Nostalrius: c'est la moindre des choses d'utiliser un peu 'spell_proc_event' non ?
```

Comment:
Yeah I was confused about what this means. I took it as more of a conversational note rather than explaining what the code was doing, so I wasn't sure whether to translate it at all or just leave it as-is